### PR TITLE
Updates from meeting.

### PR DESCRIPTION
**Pull request by odow**
- add schedule
- remove call for talks (I've already changed the Google form)

### CODE REVIEW

File: meetings/bordeaux2018/index.md [hunk 1]
```diff
@@ -24,23 +24,56 @@ The JuMP-dev workshop is a great opportunity to meet other people working on and
 
 ## Location
 
-The workshop will be held at the Institut de Mathématiques de Bordeaux, University of Bordeaux.
+The workshop will be held at the [Institut de Mathématiques de Bordeaux, University of Bordeaux](https://goo.gl/maps/PaeJbcRC58K2). If you are not staying within walking distance, the Talence campus is a short tram ride from the city centre.
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: Maybe say more explicitly that there are options to stay in Talence (walking distance) or city center (short tram ride).

File: meetings/bordeaux2018/index.md [hunk 2]
```diff
@@ -24,23 +24,56 @@ The JuMP-dev workshop is a great opportunity to meet other people working on and
...
-## Call for participants and talks
+## Call for participants
 
 The workshop is open for anybody to attend. In particular, we invite new contributors and those who have not met the core development team.
 
-Participants are encouraged to give a short (~20-minute) talk. The committee is looking for talks that will start off discussions during the workshop and beyond and for talks that present developments that the community may not be aware of. Talks will optionally be recorded and posted for those who cannot attend. We encourage speakers to give direct feedback on JuMP. See [last year's talks](https://www.youtube.com/watch?v=esOe5saQRKY&list=PLzK_rUGmc3o6EwPOCUCvBAbMJeYBS8PyY) for a flavor of the workshop.
-
-**[Use this form to indicate your participation and submit talk proposals](https://goo.gl/forms/tUeBUY6uAnAo8v5m2)**
-by **April 7th ~~March 15th~~**. There is no penalty for late sign-ups, but we may not be able to accommodate late talk proposals.
+The call for talks has now closed. However, if you wish to attend without giving a talk, **[use this form to indicate your participation](https://goo.gl/forms/tUeBUY6uAnAo8v5m2)**.
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: Could you revert the changes to the form? We're still waiting for a couple abstracts and I had asked authors to use that form (I should've mentioned this). We can say in the form description that the call for talks is closed.

File: meetings/bordeaux2018/index.md [hunk 3]
```diff
@@ -24,23 +24,56 @@ The JuMP-dev workshop is a great opportunity to meet other people working on and
...
+
+### Friday 29 June
+
+<tt>10:00</tt> Talks
+
+<tt>12:30</tt> Lunch
+
+<tt>15:00</tt> JuMP Tutorial
+
+The tutorial will cover how to use the new version of JuMP with [MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl). The tutorial will be open to anyone, even if they have not attended the workshop.
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: (no JuMP experience required)

File: meetings/bordeaux2018/index.md [hunk 4]
```diff
@@ -24,23 +24,56 @@ The JuMP-dev workshop is a great opportunity to meet other people working on and
...
+
+<tt>13:00</tt> Welcome and Introduction
+
+<tt>13:30</tt> Talks
+
+<tt>17:00</tt> Finish
+
+### Thursday 28 June
+
+<tt>10:00</tt> An Introduction to [MathOptInterface](https://github.com/JuliaOpt/MathOptInterface.jl)
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: Let's not yet commit on the starting time and the exact layout for the MOI talk.

Comment (mlubin) [reply meetings/bordeaux2018/index.md]: For Thursday I would just say "workshop day" and mention the dinner. Same for Friday except the tutorial.

File: meetings/bordeaux2018/index.md (hunk 2 above)
Comment (odow) [reply meetings/bordeaux2018/index.md]: I have reverted the form.

File: meetings/bordeaux2018/index.md [hunk 5]
```diff
@@ -24,23 +24,45 @@ The JuMP-dev workshop is a great opportunity to meet other people working on and
...
-
-**[Use this form to indicate your participation and submit talk proposals](https://goo.gl/forms/tUeBUY6uAnAo8v5m2)**
-by **April 7th ~~March 15th~~**. There is no penalty for late sign-ups, but we may not be able to accommodate late talk proposals.
+The call for talks has now closed. However, if you wish to attend without giving a talk, **[use this form to indicate your participation](https://goo.gl/forms/tUeBUY6uAnAo8v5m2)**.
 
 ## Schedule
 
-The schedule will be posted closer to the date of the workshop. We expect to have a mix of talks from participants and the core development team. Similarly to last year, the workshop will end with unstructured collaboration time to work on personal projects and brainstorm the future of JuMP.
+A preliminary schedule is outlined below. It will be updated once speakers are confirmed. The scheduled start of the workshop is at 13:00 on Wednesday, 27th June. This is to allow international visitors to arrive in Bordeaux on Wednesday morning.
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: Nit: French visitors may also choose to arrive in Bordeaux on Wednesday morning

File: meetings/bordeaux2018/index.md [hunk 6]
```diff
@@ -24,23 +24,45 @@ The JuMP-dev workshop is a great opportunity to meet other people working on and
...
+<tt>Evening</tt> Workshop Dinner
+
+*We are trying to arrange funding for the dinner, however if this is not possible, there may be a charge for participants.*
+
+
+### Friday 29 June
+
+<tt>Morning</tt> Workshop
+
+<tt>15:00</tt> JuMP Tutorial
```
Comment (mlubin) [inline meetings/bordeaux2018/index.md]: People may infer from this that nothing's going on Friday afternoon besides the JuMP tutorial. Maybe say "JuMP tutorial/ collaboration time"?

**Comment by odow** [comment]
If there are no other changes, I'll merge this tomorrow morning (NZ time, unless it gets merged before then) and then send out the emails.
